### DISCUSSION TITLE
CH02_03. jenkins pipeline 실습#1 (인프라 구축)

### DISCUSSION THREAD

**Discussion by keinetwork** [tedilabs/fastcampus-devops#85] [Q&A] [answered]
에러나요
실습1부터 안되요
cd infra/apne2/network/vpc
----------------------------
terraform init
Initializing modules...

Initializing the backend...
Error refreshing state: AccessDenied: Access Denied
	status code: 403, request id: C4PDDJYSCM5EP2WB, host id: jvIy6KJN805WZjnuVL7cyV+x+64un90OMG371a42tOVymhI/vnj9O4tQZ1mFgPCSb0TFB0RtHWQ=

**Comment by posquit0** [comment, marked as answer]
@keinetwork
안녕하세요, 다음 정보도 함께 전달해주시면 문제 해결에 도움드리기 수월할 것 같습니다.

- 수강중인 강의 패키지명
- 이슈가 있는 강의 / 실습에 대한 정보
- tedilabs/fastcampus-devops 레포지토리 내 이슈가 있는 파일 경로
- `terraform version` 명령어 출력 결과
- 실습 운영체제 정보

감사합니다.
